### ISSUE TITLE
Question on OKVQA result in the project

### ISSUE THREAD

**Issue by leeds1219** [LinWeizheDragon/Retrieval-Augmented-Visual-Question-Answering#58]
There are two recall metric or OKVQA

OKVQA Recall@5 | OKVQA Pseudo Recall@5

I understood that OKVQA training set does not have GT document annotation,

What is the Recall@5 here? The score of Recall@5 is lower than PRR@

so does this mean that OKVQA has GT document annotations for test set?

**Comment by LinWeizheDragon** [comment]
The corpus used was from another paper. You can find the paper citation in our paper. The "ground-truth" (though not actually gold, still filtered by rules) annotation was released by the authors of that paper.

**Comment by leeds1219** [comment]
Thank you for your prompt response! I appreciate the clarification.
